### PR TITLE
tests: lib: installers: added helper function "install_dovecot" to in…

### PR DESCRIPTION
**Pull request by premandfriends**
…stall dovecot server

	- Helper function will install Dovecot core server which runs both POP3 and IMAP

Signed-off-by: Prem Kumar K N <prekumar.contractor@libertyglobal.com>

### CODE REVIEW

File: tests/lib/installers.py [hunk 1]
```diff
@@ -358,6 +358,22 @@ def install_IRCserver(device):
         device.expect(device.prompt)
     except:
         device.expect(device.prompt)
+        device.sendline('apt-get install update-inetd -y') # Update inetd before installation
+        device.expect(device.prompt, timeout=90)
```
Comment (mattsm) [inline tests/lib/installers.py]: You meant to include this as well? These make sense as two separate commits or at least add the fact you are fixing an issue with IRC installation as well in the commit message.

Comment (premandfriends) [reply tests/lib/installers.py]: Update the commit message Matt . Hope this makes sense now.

Comment (mattsm) [reply tests/lib/installers.py]: No the commit does not make sense, it says nothing about why we are updating the IRC function.

Also if you could make the lines 80 chars or less....

**Comment by mattsm** [comment]
I'm not sure you meant to add that new commit?

File: tests/lib/installers.py (hunk 1 above)
Comment (premandfriends) [reply tests/lib/installers.py]: Sorry for the Mess Matt. I have address your comments and combined the 2 changes and committed.
https://github.com/lgirdk/boardfarm/commit/185b20dfd7d20827af346905fe5116ee24c4cd0a
